ix-dev/community/rust-desk/questions.yaml

```diff
@@ -100,6 +110,59 @@ questions:
           schema:
             type: boolean
             default: false
+        - variable: webui_port
+          label: Port for the webui
+          description: |
+            This port will be used to connect to the webui.</br>
+            The internal port will always be 21114.</br>
+            This is only available in the pro version.
+          schema:
+            type: dict
+            show_if: [["host_network", "=", false],["rust_desk.edition", "=", "pro"]]
+            attrs:
+              - variable: bind_mode
+                label: Port Bind Mode
+                description: |
+                  The port bind mode.</br>
+                  - Publish: The port will be published on the host for external access.</br>
+                  - Expose: The port will be exposed for inter-container communication.</br>
+                  - None: The port will not be exposed or published.</br>
+                  Note: If the Dockerfile defines an EXPOSE directive,
+                  the port will still be exposed for inter-container communication regardless of this setting.
+                schema:
+                  type: string
+                  default: "published"
```

Comment:
default to `""` and if a user selects the pro, will also have to publish this port